content/en/users/security/secrets-store/gui/_index.md

```diff
@@ -0,0 +1,80 @@
+---
+title: Secrets Store Web Interface
+linkTitle: Web Interface
+type: docs
+weight: 10
+description: >
+  The web interface of EGI Secrets Store
+---
+
+Although the recommended way to access and use EGI Secrets Store is via the
+[command-line interface](../cli), the service can also be accessed via its
+web interface.
+
+{{% alert title="Tip" color="info" %}} Tutorials about using the web interface
+are available on the
+[Hashicorp Vault site](https://developer.hashicorp.com/vault/tutorials/getting-started-ui).
+{{% /alert %}}
+
+## Using the web interface
+
+Follow the steps below to access the web interface of EGI Secrets Store:
+
+<!-- markdownlint-disable no-bare-urls -->
+- Open https://vault.services.fedcloud.eu:8200 in your browser.
```

Comment:
I would suggest to remove the `no-bare-urls` directive, and either use a proper link name, use the URL as the name, or put it into backticks (not preferred as it won't provide a clickable link)

Reply:
Backicks is a bad idea here. I am against blindly following rules, even when they do not make sense. Repeating an URL unnecessarily just for the sake of a rule is wrong. Next time this URL changes, you have to update it twice.